### ISSUE TITLE
Do you have any plan to upgrade fork-ts-checker-webpack-plugin to v5

### ISSUE THREAD

**Issue by tieppt** [nrwl/nx#3492]
## Description

https://github.com/TypeStrong/fork-ts-checker-webpack-plugin

For now, we're using version 3.1.1 of this plugin. It does not report type checking result (error) to devServer. But if I upgrade to v5 with custom WebPack config, it can work as expected.

**Comment by Sikora00** [comment]
I raise the question. I spent a lot of time to debug why each of my NestJS microservices consumes 3GB of RAM during development. Each has 6 workers with 2GB memory limit. As I checked fast in newer versions of the fork-ts-checker-webpack-plugin they reduced workers number by default and improved the performance of the single worker.

**Comment by asadullah-10p** [comment]
`fork-ts-checker-webpack-plugin@3.1.1` is used in following nrwl packages:
```shell
@nrwl/cypress@11.0.15
│ └── fork-ts-checker-webpack-plugin@3.1.1 
├─┬ @nrwl/express@11.0.16
│ └─┬ @nrwl/node@11.0.16
│   └── fork-ts-checker-webpack-plugin@3.1.1  deduped
├─┬ @nrwl/node@11.0.2
│ └── fork-ts-checker-webpack-plugin@3.1.1  deduped
├─┬ @nrwl/react@11.0.16
│ ├─┬ @nrwl/cypress@11.0.16
│ │ └── fork-ts-checker-webpack-plugin@3.1.1  deduped
│ └─┬ @nrwl/web@11.0.16
│   └── fork-ts-checker-webpack-plugin@3.1.1  deduped
└─┬ @nrwl/web@11.0.15
  └── fork-ts-checker-webpack-plugin@3.1.1  deduped
```

Webpack 5 is not working well with  fork-ts-checker-webpack-plugin@3.1.1 and is crashing builds. Folks from fork-ts-checker-webpack-plugin [have fixed this in v6](https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/issues/540).
Since nrwl is still using the old version of fork-ts-checker-webpack-plugin, my team is unable to update our project to webpack 5. 

@nrwl Any plans to update this dep?

**Comment by ultd** [comment]
Can we please get update on this? I continue to get the following error:

```
ℹ ｢wdm｣: Compiling...
events.js:292
      throw er; // Unhandled 'error' event
      ^

Error [ERR_IPC_CHANNEL_CLOSED]: Channel closed
    at ChildProcess.target.send (internal/child_process.js:679:16)
    at RpcProvider._dispatch (/usr/src/app/node_modules/fork-ts-checker-webpack-plugin/lib/cluster.js:42:20)
    at RpcProvider.rpc (/usr/src/app/node_modules/worker-rpc/lib/RpcProvider.js:32:14)
    at /usr/src/app/node_modules/fork-ts-checker-webpack-plugin/lib/cluster.js:53:83
    at Array.map (<anonymous>)
    at Object.<anonymous> (/usr/src/app/node_modules/fork-ts-checker-webpack-plugin/lib/cluster.js:53:56)
    at Generator.next (<anonymous>)
    at /usr/src/app/node_modules/fork-ts-checker-webpack-plugin/lib/cluster.js:7:71
    at new Promise (<anonymous>)
    at __awaiter (/usr/src/app/node_modules/fork-ts-checker-webpack-plugin/lib/cluster.js:3:12)
Emitted 'error' event on ChildProcess instance at:
    at internal/child_process.js:683:35
    at processTicksAndRejections (internal/process/task_queues.js:79:11) {
  code: 'ERR_IPC_CHANNEL_CLOSED'
}
```

The bug is fixed in `v4.0.4` as mentioned [here](https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/issues/363#issuecomment-585601162)

Hope we can get movement on this issue ASAP.
